src/SDKs/DataLake.Store/Management.DataLake.Store/generate.cmd

```diff
@@ -4,17 +4,5 @@
 ::
 
 @echo off
-setlocal
-
-set accountSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/156cd442edf6badb940149878cecfba2d3198776/arm-datalake-store/account/2016-11-01/swagger/account.json"
-set filesystemSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/156cd442edf6badb940149878cecfba2d3198776/arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json"
-
-set repoRoot=%~dp0..\..\..\..
-set generateFolder=%~dp0Generated
-
-if exist %generateFolder% rd /S /Q  %generateFolder%
-
-call autorest --latest -CodeGenerator Azure.CSharp -Input %accountSpecFile% -Namespace Microsoft.Azure.Management.DataLake.Store -outputDirectory %generateFolder% -Header MICROSOFT_MIT -ClientSideValidation %~5
-call autorest --latest -CodeGenerator Azure.CSharp -Input %filesystemSpecFile% -Namespace Microsoft.Azure.Management.DataLake.Store -outputDirectory %generateFolder% -Header MICROSOFT_MIT -ClientSideValidation %~5
-
-endlocal
\ No newline at end of file
+call %~dp0..\..\..\..\tools\generate.cmd datalake-store/resource-manager %*
+call %~dp0..\..\..\..\tools\generate.cmd datalake-store/data-plane %*
```

Comment:
minor: put in a new line at the end of the file (it was missing previously but we should standardize the empty new line since you are touching the file)